Frameworks/CoreGraphics/CGDiscardableImage.mm

```diff
@@ -105,9 +109,22 @@
     return _forward->BytesPerPixel();
 }
 
+int CGDiscardableImageBacking::BitsPerComponent() {
+    ConstructIfNeeded();
```

Comment:
Once we construct the image once, we should probably cache all of these properties, so if the bitmap is discarded, we can get the info without reconstructing it.  I see that the pattern is preexisting though, can the surface change after the image is discarded (and then re-used?).